File: Proyectos/Android/ExamenFragments_DavidCarroSalinas/app/src/main/java/com/example/examenfragments_davidcarrosalinas/Modelo/Usuario.java
package com.example.examenfragments_davidcarrosalinas.Modelo;

import java.io.Serializable;

public class Usuario implements Serializable {
    private String user;
    private String pass;
    private String rol;

    public Usuario(String user, String pass, String rol) {
        this.user = user;
        this.pass = pass;
        this.rol = rol;
    }

    public String getUser() {
        return user;
    }

    public void setUser(String user) {
        this.user = user;
    }

    public String getPass() {
        return pass;
    }

    public void setPass(String pass) {
        this.pass = pass;
    }

    public String getRol() {
        return rol;
    }

    public void setRol(String rol) {
        this.rol = rol;
    }

    @Override
    public String toString() {
        return "Usuario{" +
                "user='" + user + '\'' +
                ", pass='" + pass + '\'' +
                ", rol='" + rol + '\'' +
                '}';
    }
}

File: Desarrollo multimedia y dispositivos móviles/UT2/Teoría/recursosExamenAndroidDiciembre/kotlin/Usuario.kt
package com.example.examenfragmentskotlin

import java.io.Serializable

class Usuario : Serializable {
    internal lateinit var user: String
    private lateinit var pass: String
    private lateinit var rol: String

    constructor (user: String, pass: String, rol: String) {
        this.user = user
        this.pass = pass
        this.rol = rol
    }

    fun getRol() : String {
        return rol;
    }

    override fun toString(): String {
        return "Usuario{" +
                "user='" + user + '\'' +
                ", pass='" + pass + '\'' +
                ", rol='" + rol + '\'' +
                '}'
    }
}